### ISSUE TITLE
Is the site down?

### ISSUE THREAD

**Issue by github-account1111** [JubbeArt/removeddit#42]

**Comment by wyatt8740** [comment]
Seems to be, yes.

**Comment by ShadowJonathan** [comment]
@JubbeArt could you look at this, please?

**Comment by ShadowJonathan** [comment]
For anyone who needs a site which works similar to this one, use reveddit.com

**Comment by Scientist07** [comment]
> 
> 
> For anyone who needs a site which works similar to this one, use reveddit.com

Doesn't show comments deleted by user though. That feature is what I use most as I often check back on people I've banned.

**Comment by spmedia** [comment]
I just emailed OP on two emails I found for him to see why its down and whats going on. I hope he comes back, brings the site online, and then pushes all the stuffs in the pull requests. Seems weird to be down for so long. He should throw some ads on it or something to at least monetize it and make it worth his time.

**Comment by aplotor** [comment]
> He should throw some ads on it or something to at least monetize it and make it worth his time.

it seems to be quite difficult to get reddit's approval to monetize any app using reddit data.

**Comment by Scientist07** [comment]
> 
> 
> I just emailed OP on two emails I found for him to see why its down and whats going on. I hope he comes back, brings the site online, and then pushes all the stuffs in the pull requests. Seems weird to be down for so long. He should throw some ads on it or something to at least monetize it and make it worth his time.

He ever responded to the emails?

**Comment by spmedia** [comment]
@Scientist07  nope :\

**Comment by spmedia** [comment]
@JubbeArt beep boop beep ping ping put site back online

**Comment by mustafa-archived** [comment]
I hope he's ok

**Comment by gurnec** [comment]
> I hope he's ok

We all do, I'm sure.

I should apologize for being a bit spammy, but I'm going to do it anyways. While Removeddit.com is down, people are free to try https://www.unddit.com/ instead. It's mostly a clone of Removeddit, along with a few updates. The source is a fork of this repo, and can be viewed under my account.

Cheers!

**Comment by Scientist07** [comment]
> 
> 
> > I hope he's ok
> 
> We all do, I'm sure.
> 
> I should apologize for being a bit spammy, but I'm going to do it anyways. While Removeddit.com is down, people are free to try https://www.unddit.com/ instead. It's mostly a clone of Removeddit, along with a few updates. The source is a fork of this repo, and can be viewed under my account.
> 
> Cheers!

You're a lifesaver man.

**Comment by github-account1111** [comment]
>I hope he's ok

FWIW he has had activity after I created this issue so I'd say so?

@Scientist07 Thanks so much. Just to make sure I'm understanding correctly: there isn't some sort of a database of reddit posts that's been lost because this repo's author went MIA, correct? In other words, these are all just frontends for Jason Baumgartner's database mentioned on that page?

**Comment by Scientist07** [comment]
> 
> 
> > I hope he's ok
> 
> FWIW he has had activity after I created this issue so I'd say so?
> 
> @Scientist07 Thanks so much. Just to make sure I'm understanding correctly: there isn't some sort of a database of reddit posts that's been lost because this repo's author went MIA, correct? In other words, these are all just frontends for Jason Baumgartner's database mentioned on that page?

Don't thank me. Thank the guy who linked his version. As far as I understand it, yes that's how it works. Nothing was lost.

**Comment by gurnec** [comment]
@github-account1111 Yes, @Scientist07 is correct in that Removeddit's currently-missing state did not affect the underlying data stored by Pushshift.io.

With that said, there is an issue with Pushshift that started sometime in August—comments (but not posts I believe) that are archived by Pushshift, and then later removed/deleted within about<sup>*</sup> 2.5 days are removed from the Pushshift archive at that 2.5 day mark. I don't think it's known if this is intentional or a bug, nor known if this removal is permanent or temporary (if a bug gets fixed).

When this happens, you'll see a message like `removed(?) from archive after 2 days` on Unddit. You can read more about it [on Reddit here](https://www.reddit.com/r/pushshift/comments/pgzdav/the_api_now_appears_to_rewrite_nearly_all/) (that post author is Removeddit's developer btw).

Speaking of Removeddit, because they operate their own archival service (I don't know much about it), you can sometimes see comments that have been removed from Pushshift/Unddit over there.

\* This time delay has been slowly increasing since August, I think it started at 1 day.

**Comment by github-account1111** [comment]
> 
> 
> > > I hope he's ok
> > 
> > 
> > FWIW he has had activity after I created this issue so I'd say so?
> > @Scientist07 Thanks so much. Just to make sure I'm understanding correctly: there isn't some sort of a database of reddit posts that's been lost because this repo's author went MIA, correct? In other words, these are all just frontends for Jason Baumgartner's database mentioned on that page?
> 
> Don't thank me. Thank the guy who linked his version. As far as I understand it, yes that's how it works. Nothing was lost.

Ah yea my bad I meant to @gurnec :D

>Speaking of Removeddit, because they operate their own archival service (I don't know much about it), you can sometimes see comments that have been removed from Pushshift/Unddit over there.

Wait, so there *is* some sort of database other than Pushshift Removeddit was pulling from? Does only the owner have access to it?

**Comment by rollingmoai** [comment]
@gurnec Thanks for the fork. Can you open the issues tab on your repo since it is closed?

**Comment by gurnec** [comment]
@itsmeboy0 Done. I didn't even realize they were disabled, thanks.
